packages/las-sdk-core/src/client.spec.ts

```diff
@@ -206,10 +221,14 @@ describe('Documents', () => {
 
   describe('listDocuments', () => {
     test('valid request', async () => {
-      const testBatchId = uuidv4();
-      const listDocumentsPromise = client.listDocuments({ batchId: testBatchId });
-      await expect(listDocumentsPromise).resolves.toBeDefined();
-    });
+      test.each<[ListDocumentsOptions]>([
+        [{ batchId: batchId() }],
+        [{ consentId: consentId() }],
+        [{ datasetId: datasetId() }],
+      ])('parameters: %s', async options => {
+        const listDocumentsPromise = client.listDocuments(options);
+        await expect(listDocumentsPromise).resolves.toBeDefined();
```

Comment:
I'm wondering if the `.toBeDefined()` does much here. If we're only checking if the request succeeds and thus resolves, I think maybe `await expect(listDocumentsPromise).resolves` is enough (and clearer)?

Reply:
Would it be the same for .rejects?

Reply:
I think so (not 100% sure if these work that way BTW... just a thought)